README.md

```diff
@@ -120,15 +120,15 @@ gem 'arduino_ci', path: '/path/to/development/dir/for/arduino_ci'
 ### Installing the Dependencies
 
 Fulfilling the `arduino_ci` library dependency is as easy as running either of these two commands:
-```
+
+```sh
```

Comment:
This should be `console`, not `sh`, because this code block includes the prompt characters.  The difference in handling is easier to see when the prompt itself has some formatting:

Code block with `sh` formatting
```sh
user@host:~$ bundle install   # adds packages to global library (may require admin rights)
user@host:~$ bundle install --path vendor/bundle   # adds packages to local library
```

Code block with `console` formatting
```console
user@host:~$ bundle install   # adds packages to global library (may require admin rights)
user@host:~$ bundle install --path vendor/bundle   # adds packages to local library
```

Reply:
Nice, I did not know about the console option. Looks good.